src/redux/migration.ts

```diff
@@ -16,6 +16,13 @@ export const migrateEachYear = <S extends USTState>(state: S): S =>
       }
     })
 
+    // Ensure interestIncome on K1s
+    acc[year].scheduleK1Form1065s.forEach((k1) => {
```

Comment:
I didn't notice it in the w2s part above, but this is modifying the state object in place, and then returning it. I think in redux this will cause a rerendering problem if there is something depending on a subscription of the `scheduleK1Form1065s` array. Is that right?